website/docs/reference/warehouse-setups/dremio-setup.md

```diff
@@ -47,30 +47,257 @@ pip is the easiest way to install the adapter:
- dbt-dremio
-
+ 
 Follow the repository's link for os dependencies.
 
-## Connecting to Dremio with **dbt-dremio**
+## Prerequisites for Dremio Cloud
+Before connecting from project to Dremio Cloud, follow these prerequisite steps:
+* Ensure that you have the ID of the Sonar project that you want to use. See [Obtaining the ID of a Project](https://docs.dremio.com/cloud/cloud-entities/projects/#obtaining-the-id-of-a-project).
+* Ensure that you have a personal access token (PAT) for authenticating to Dremio Cloud. See [Creating a Token](https://docs.dremio.com/cloud/security/authentication/personal-access-token/#creating-a-token).
+
+
+## Prerequisites for Dremio Software
+
+* Ensure that you are using version 22.0 or later.
+* Enable these support keys in your Dremio cluster:
+  * `dremio.iceberg.enabled`
+  * `dremio.iceberg.ctas.enabled`
+  * `dremio.execution.support_unlimited_splits`
+
+  See <a target="_blank" href="https://docs.dremio.com/software/advanced-administration/support-settings/#support-keys">Support Keys</a> in the Dremio documentation for the steps.
+* If you want to use TLS to secure the connection between dbt and Dremio Software, configure full wire encryption in your Dremio cluster. For instructions, see <a target="_blank" href="https://docs.dremio.com/software/deployment/wire-encryption-config/">Configuring Wire Encryption</a>.
+
 
-### Connecting with ZooKeeper
+## Initializing a Project
 
-I have no means to test [connection with ZooKeeper](https://docs.dremio.com/drivers/dremio-connector.html#connecting-to-zookeeper). 
-If you do need this, contact me and I will provide you with a branch you can test.
+1. Run the command `dbt init <project_name>`.
+2. Select `dremio` as the database to use.
+3. Select one of these options to generate a profile for your project:
+    * `dremio_cloud` for working with Dremio Cloud
+    * `software_with_username_password` for working with a Dremio Software cluster and authenticating to the cluster with a username and a password
+    * `software_with_pat` for working with a Dremio Software cluster and authenticating to the cluster with a personal access token
+4. Append these lines to the end of the content of the `dbt_project.yml` file at the root of your project directory:
+```
+vars:
+    dremio:reflections_enabled: false
+```
 
-### Direct connection to a coordinator
+Next, configure the profile for your project.
 
+## Profiles
+
+When you initialize a project, you create one of these three profiles. You must configure it before trying to connect to Dremio Cloud or Dremio Software.
+
+* [Profile for Dremio Cloud](#cloud)
+* [Profile for Dremio Software with Username/Password Authentication](#software_up)
+* [Profile for Dremio Software with Authentication Through a Personal Access Token](#software_pat)
+
+For descriptions of the configurations in these profiles, see [Configurations](#configurations).
+
+<a id="cloud"></a>
+### Profile for Dremio Cloud
```

Comment:
@rheath-dremio extra credit opportunity you want to turn the 3 profile options into tabs? Here's an example that uses Docusaurus's `Tab` and `File` elements.
https://github.com/dbt-labs/docs.getdbt.com/blob/a4399bfad96d209806de66332e7a69198caf8bb5/website/docs/reference/warehouse-setups/mssql-setup.md?plain=1#L103-L130

Reply:
Extra credit? I think I'm already an A student, but I'll take it. Done.